src/clr-core/.storybook/public/demo.css

```diff
@@ -0,0 +1,65 @@
+/* Temporary Clarity typography for demos until core global styles are complete */
+body {
+  font-family: 'Clarity City', 'Avenir Next', 'Helvetica Neue', Arial, sans-serif;
+}
+
+.doc {
+  color: hsl(0, 0%, 40%);
+  font-weight: 400;
+  font-size: 0.7rem;
+  letter-spacing: normal;
+  line-height: 1.2rem;
+}
+
+h1,
+h2 {
+  color: hsl(0, 0%, 40%);
+  font-weight: 200;
+  font-size: 1.6rem;
+}
+
+h2 {
+  font-size: 1.4rem;
+}
+
+p {
+  color: hsl(0, 0%, 40%);
+  font-size: 0.7rem;
+}
```

Comment:
These are some temporary basic styles for typography until we can plan how core users should consume clarity typography styles.